trade-finance-manager-api/src/graphql/reducers/mappings/gef-facilities/mapGefFacilityDates.api-test.js

```diff
@@ -72,6 +72,13 @@ describe('mapGefFacilityDates', () => {
     expect(result.tenor).toEqual(expected);
   });
 
+  it('should not map coverStartDate when date is null', () => {
+    mockFacility.coverStartDate = null;
+    const result = mapGefFacilityDates(mockFacility, mockFacilityTfm, MOCK_GEF_DEAL);
+    // undefined as not set in function
+    expect(result.coverStartDate).toEqual(undefined);
```

Comment:
can we try here to use `.toBeUndefined()` ?

Reply:
done